linalg/sundials.cpp

```diff
@@ -2184,6 +2188,43 @@ void KINSolver::SetMAA(int m_aa)
    }
 }
 
+void KINSolver::SetDamping(real_t beta)
+{
+   if (sundials_mem)
+   {
+      flag = KINSetDamping(sundials_mem, beta);
+      MFEM_ASSERT(flag == KIN_SUCCESS, "error in KINSetDamping()");
+   }
+}
+
+void KINSolver::SetDampingAA(real_t beta)
+{
+   if (sundials_mem)
+   {
+      flag = KINSetDampingAA(sundials_mem, beta);
+      MFEM_ASSERT(flag == KIN_SUCCESS, "error in KINSetDampingAA()");
+   }
+}
+
+void KINSolver::SetDelayAA(int delay)
+{
+   if (sundials_mem)
+   {
+      flag = KINSetDelayAA(sundials_mem, delay);
+      MFEM_ASSERT(flag == KIN_SUCCESS, "error in KINSetDelayAA()");
+   }
+}
```

Comment:
I have the same concern here as for `KINSolver:SetDamping` (https://github.com/mfem/mfem/pull/4419/files#r1720392394)